mantidproject/mantid
# Release notes 4.2 final fixes

**Description of work.**
Diffraction page - remove Duplicate Single Crystal section
Index page - small reword
mantidworkbench page - add a known issue along with relevant image

**To test:**
Build docs.html and check described changes work as desired.

*This does not require release notes* because **It is release notes**


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
